src/contexts/IdContext.tsx

```diff
@@ -37,34 +37,34 @@ export const IdContextProvider = ({ children }: { children: ReactNode }) => {
     sessionStorage.getItem('earthId') ?? ''
   )
 
+  const [continentId, setContinentId] = useState<string>(
+    sessionStorage.getItem('continentId') ?? ''
+  )
+
+  const [countryId, setCountryId] = useState<string>(
+    sessionStorage.getItem('countryId') ?? ''
+  )
+
+  const [campusId, setCampusId] = useState<string>(
+    sessionStorage.getItem('campusId') ?? ''
+  )
+
   const setRefId = (earthId: string) => {
     setEarthId(earthId)
     sessionStorage.setItem('earthId', earthId)
```

Comment:
I think this would be better if it said planetId

Reply:
@dabick14

Reply:
Ok. Makes sense.